### DISCUSSION TITLE
Adding support for the Canary keyboard layout

### DISCUSSION THREAD

**Discussion by yoshuawuyts** [manna-harbour/miryoku#345] [Q&A] [answered]
[Canary](https://github.com/Apsu/Canary) is a decently popular keyboard layout, released around two years ago. I'm interested in adding a Miryoku layout for this, and I was wondering whether there would be any interest in this being upstreamed? Thanks!

**Comment by manna-harbour** [comment, marked as answer]
You can add it via custom_config.h, and share your config. See https://github.com/manna-harbour/miryoku/discussions/85.

After "Auto generate miryoku_layer_selection.h or replace with make" (https://github.com/manna-harbour/miryoku/issues/90) is implemented, new alpha arrangement options may be added, but they won't be added now as the implementation is being maintained manually at the moment.
